src/relayedTransactionV1Builder.spec.ts

```diff
@@ -74,4 +75,84 @@ describe("test relayed v1 transaction builder", function () {
         assert.equal(relayedTxV1.getData().toString(), "relayedTx@7b226e6f6e6365223a3139382c2273656e646572223a2267456e574f65576d6d413063306a6b71764d354241707a61644b46574e534f69417643575163776d4750673d222c227265636569766572223a22414141414141414141414141415141414141414141414141414141414141414141414141414141432f2f383d222c2276616c7565223a302c226761735072696365223a313030303030303030302c226761734c696d6974223a36303030303030302c2264617461223a225a3256305132397564484a68593352446232356d6157633d222c227369676e6174757265223a2239682b6e6742584f5536776674315464437368534d4b3454446a5a32794f74686336564c576e3478724d5a706248427738677a6c6659596d362b766b505258303764634a562b4745635462616a7049692b5a5a5942773d3d222c22636861696e4944223a2256413d3d222c2276657273696f6e223a317d");
         assert.equal(relayedTxV1.getSignature().hex(), "c7d2c3b971f44eca676c10624d3c4319f8898af159f003e1e59f446cb75e5a294c9f0758d800e04d3daff11e67d20c4c1f85fd54aad6deb947ef391e6dd09d07");
     });
+
+    it("should compute guarded inner Tx - relayed v1 transaction", async function () {
+        const networkConfig = {
+            MinGasLimit: 50_000,
+            GasPerDataByte: 1_500,
+            GasPriceModifier: 0.01,
+            ChainID: "T"
+        };
+
+        const innerTx = new Transaction({
+            nonce: 198,
+            sender: bob.address,
+            receiver: Address.fromBech32("erd1qqqqqqqqqqqqqpgq54tsxmej537z9leghvp69hfu4f8gg5eu396q83gnnz"),
+            gasLimit: 60000000,
+            chainID: networkConfig.ChainID,
+            data: new TransactionPayload("getContractConfig"),
+            guardian: grace.address,
+            version: TransactionVersion.withTxOptions(),
+            options: TransactionOptions.withTxGuardedOptions(),
+        });
+
+        await bob.signer.sign(innerTx);
+        innerTx.applyGuardianSignature(innerTx.getSignature());
```

Comment:
2 ways of refactoring:
1. use a real case example in this unit test. That is, a real guardian that performs the signing that would result in the guardian signature
2. use a mock signature here instead of signing the transaction and applying the regular signature as the guardian signature

Reply:
second choice